src/components/TheHeader.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <section>
+    <article class='bone-yard-header'>
+      <h1>Bone Yard</h1>
+      <img class='bone-yard-logo' alt='Bone Yard pawprint logo' src='https://images-na.ssl-images-amazon.com/images/I/31jVeYUkbhL._AC_SY550_.jpg'/>
+    </article>
+    <nav>
+      <router-link to='/'>home</router-link> <br />
+			<!-- might not need the home link here - click on bone yard title to go home -->
+      <router-link to='/my-parks'>my parks</router-link> <br/>
+			<router-link to='/search'>find park</router-link>
+    </nav>
```

Comment:
Looking at this again, with a mobile first approach this makes it look good! However, setting a `<br />` for each `router-link` for desktop may have some issues. We may want to assign a specific class for the `nav` and control it with flex and media queries. This way, we have more control over when the links have to go vertical and when they can go back to horizontal.

Reply:
100% agree! I figure we'll take our those `<br />` tags when we get to styling more intentionally.  Definitely not a long-term solution by any means.

Reply:
We'd probably just set a `display: block` for the `nav` so that each element within takes up its own space horizontally.